### ISSUE TITLE
option for interactive password entry

### ISSUE THREAD

**Issue by Mikhail-Izioumtchenko** [sjmudd/ps-top#15]
Hi Simon,

it would be nice to have an option for invisible interactive password entry as in mysql -p

**Comment by sjmudd** [comment]
Hi Mikhail,

This should be fixed in `v1.0.11` via the command line option `--askpass`

e.g.
```
$ ./ps-top --socket=/tmp/mysql.sock --user=root --askpass
Password:

**Comment by sjmudd** [comment]
Let. me know if this works for you.
